docs/build_docs.sh

```diff
@@ -78,4 +80,28 @@ while getopts "piez" opt; do
 done
 
 # use 'bundle exec' to insert the local Ruby dependencies
-bundle exec jekyll ${JEKYLL_CMD} ${JEKYLL_CONFIG} --source "${DOCS_SRC}" --destination "${DOCS_DST}"
+
+if [ "${JEKYLL_CMD}" = "build" ] && [ -z "${JEKYLL_CONFIG}" ]; then
```

Comment:
if we modified the e/z cases above to instead set DOC_LANGUAGES to en/zh we wouldn't need this condition nor else branch

Reply:
maybe a branch for the DOC_LANGUAGES (to get all languages if none of the e/z options was used) but simpler anyway; I'll give it a try

Reply:
oh, the branch is still always needed since there is no parallel `serve`...so not sure whether this change makes sense